hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/OzoneRpcClientTests.java

```diff
@@ -4810,17 +4812,15 @@ void testMultiPartUploadWithStream(ReplicationConfig replicationConfig)
   }
 
   @Test
-  public void testUploadWithStreamAndMemoryMappedBuffer() throws IOException {
+  public void testUploadWithStreamAndMemoryMappedBuffer(@TempDir Path dir) throws IOException {
     // create a local dir
-    final String dir = GenericTestUtils.getTempPath(
-        getClass().getSimpleName());
-    GenericTestUtils.assertDirCreation(new File(dir));
+    GenericTestUtils.assertDirCreation(new File(dir.toString()));
```

Comment:
nit: `dir` is created by JUnit, isn't it?

Reply:
Correct, we can remove this check